website/docs/docs/partnerships.mdx

```diff
@@ -189,11 +183,17 @@ For partners who are new to working with Hashicorp, we recommend [signing up for
 
 ### 2. Develop & Test
 
-Partners should build an integration using the [Run task APIs in Terraform Cloud](/cloud-docs/api-docs/run-tasks). For example, [Snyk](https://docs.snyk.io/features/integrations/ci-cd-integrations/integrating-snyk-with-terraform-cloud) created a run task to automatically detect misconfigurations in the Terraform plan. To learn more about how post-plan run tasks can enhance the Terraform Cloud workflow, refer to the following diagram and read our [run tasks announcement](https://www.hashicorp.com/blog/terraform-cloud-run-tasks-beta-now-available).
+Partners should build an integration using the [run tasks APIs in Terraform Cloud](/cloud-docs/api-docs/run-tasks). To learn more about how post-plan run tasks can enhance the Terraform Cloud workflow, refer to the following diagram and read our [run tasks announcement](https://www.hashicorp.com/blog/terraform-cloud-run-tasks-beta-now-available). For a more hands-on experience with run tasks follow the [HCP Packer run tasks learn guide here](https://learn.hashicorp.com/tutorials/packer/setup-tfc-run-task?in=terraform/cloud).
+
+For the run tasks API documentation [click here](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/run-tasks/run-tasks).
+
+![RunTask Diagram](/img/docs/post-plan-runtask-diagram.png)
 
--> **Note:** Pre-plan run tasks are only available as a public beta. [Updated September 1, 2022]
+-> **Note:** Pre-plan & pre-apply run tasks are only available as a public beta. [Updated September 1, 2022]
 
-![RunTask Diagram](/img/docs/runtask-diagram.png)
+To better understand how pre-plan run task enhances the Terraform workflow, see diagram listed below. For additional API resources, [click here](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/run-tasks/run-task-stages-and-results).
```

Comment:
```suggestion
To better understand how pre-plan run task enhances the Terraform workflow, refer to the diagram of pre-plan tasks. For additional API resources, [click here](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/run-tasks/run-task-stages-and-results).
```

Suggest changing "see" to "refer to" to avoid ableist language. Suggest removing "below" to avoid directional language and to more specifically tell the user which diagram to find.